src/main/java/page_objects/ChangePasswordPage.java

```diff
@@ -0,0 +1,15 @@
+package page_objects;
+
+import helpers.BrowserHelper;
+import helpers.Constant;
+import helpers.Wait;
+import org.openqa.selenium.By;
+
+public class ChangePasswordPage extends BasePage {
+    private final By pageTitle = By.xpath("//span[contains(text(),'Change password')]//ancestor::div[@id='page']//h1");
```

Comment:
hmm, I can change 'Change password' to other text and the 'Change password' h1 is still found.
By.xpath("//div[@id='content']/h1[text()='Change password']");

Reply:
Thank you for your reviewed. I've changed to By.cssSelector("#content h1"); Please check and review it. Thank you so much!